packages/webapp/pages/posts/[id].tsx

```diff
@@ -238,7 +238,7 @@ const PostPage = ({ id, postData }: Props): ReactElement => {
 
   return (
     <>
-      <div className="flex relative flex-col flex-wrap mx-auto max-w-full">
+      <div className="flex relative flex-col flex-wrap mx-auto w-full laptopL:w-auto max-w-full">
```

Comment:
Why the `laptopL:auto`? We already set the width for this resolution in the PageContainer element

Reply:
weird case where the full class would overwrite the one we set.
Auto would just take that one and unset the full width.

Reply:
Ok!